lightstep_incident_response/manifest.json

```
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b61db30e-3cbc-4dbe-b055-824f9e46e006",
+  "app_id": "lightstep--incident-response",
```

```suggestion
  "app_id": "lightstep_incident_response",
```
Just to keep things consistent between directory name, metrics prefix, and app id